ArduCopter/APM_Config.h

```diff
@@ -21,7 +21,7 @@
 //#define FRSKY_TELEM_ENABLED   DISABLED            // disable FRSky telemetry
 //#define ADSB_ENABLED          DISABLED            // disable ADSB support
 //#define PRECISION_LANDING     DISABLED            // disable precision landing using companion computer or IRLock sensor
-//#define BEACON_ENABLED        DISABLED            // disable beacon support
+//#define BEACON_ENABLED        DISABLED            // disable localization beacon support
```

Comment:
I actually prefer the older comment, we never refer to beacon support/library as localization beacon.